### ISSUE TITLE
unicode characters create errors with pdflatex

### ISSUE THREAD

**Issue by pbatroff** [canismarko/dungeon-sheets#33]
When creating the wizard spellbook example sheet, pdflatex crashes because of unicode special characters in the python classes. 

Reproduction:
`makesheets wizard1.py` - errors are in Teleport spell, when calling pdflatex manually:

`pdflatex --output-directory $HOME/Documents/src/dungeon-sheets/examples wizard1_spellbook.tex`

Versions: 
```
pdflatex --version
pdfTeX 3.14159265-2.6-1.40.20

python --version
Python 3.8.1
```
Running under Linux.

All but two characters are special space characters 'U+2003' , see [here](https://www.compart.com/de/unicode/U+2003). When replaced pdflatex works fine. 

Here is a list of occurences:

```
dungeonsheets/spells/spells_a.py:165:    Weapons. You grow claws, fangs, spines, horns, or a different natural weapon of 
dungeonsheets/spells/spells_a.py:246:    turns, you can use your actions to transform affected creatures into new forms. 
dungeonsheets/spells/spells_a.py:260:     
dungeonsheets/spells/spells_a.py:431:     Any 
dungeonsheets/spells/spells_a.py:483:    antipathy or sympathy as the aura’s effect. 
dungeonsheets/spells/spells_a.py:487:     creatures of the kind you designated to feel an intense urge to leave the area 
dungeonsheets/spells/spells_a.py:488:    and avoid the target. When such a creature can see the target or comes within 60
dungeonsheets/spells/spells_a.py:489:     feet of it, the creature must succeed on a Wisdom saving throw or 
dungeonsheets/spells/spells_a.py:490:    become frightened. The creature remains frightened while it can see the target 
dungeonsheets/spells/spells_a.py:491:    or is within 60 feet of it. While frightened by the target, the creature must 
dungeonsheets/spells/spells_a.py:492:    use its movement to move to the nearest safe spot from which it can’t see the 
dungeonsheets/spells/spells_a.py:493:    target. If the creature moves more than 60 feet from the target and can’t see 
dungeonsheets/spells/spells_a.py:494:    it, the creature is no longer frightened, but the creature becomes frightened 
dungeonsheets/spells/spells_a.py:495:    again if it regains sight of the target or moves within 60 feet of it. 
dungeonsheets/spells/spells_a.py:499:    The enchantment causes the specified creatures to feel an intense 
dungeonsheets/spells/spells_a.py:500:    urge to approach the target while within 60 feet of it or able to see it. When 
dungeonsheets/spells/spells_a.py:501:    such a creature can see the target or comes within 60 feet o fit, the creature 
dungeonsheets/spells/spells_a.py:502:    must succeed on a Wisdom saving throw or use its movement on each of its turns 
dungeonsheets/spells/spells_a.py:503:    to enter the area or move within reach of the target. When the creature has done
dungeonsheets/spells/spells_a.py:504:     so, it can’t willingly move away from the target. If the target damages or 
dungeonsheets/spells/spells_a.py:505:    otherwise harms an affected creature, the affected creature can make a 
dungeonsheets/spells/spells_a.py:506:    Wisdom saving throw to end the effect, as described below. 
dungeonsheets/spells/spells_a.py:510:    If an affected creature ends its turn while not within 60 feet of the target 
dungeonsheets/spells/spells_a.py:511:    or able to see it, the creature makes a Wisdom saving throw. On a successful 
dungeonsheets/spells/spells_a.py:512:    save, the creature is no longer affected by the target and recognizes the 
dungeonsheets/spells/spells_a.py:513:    feeling of repugnance or attraction as magical. In addition, a creature affected
dungeonsheets/spells/spells_a.py:514:     by the spell is allowed another Wisdom saving throw every 24 hours while the 
dungeonsheets/spells/spells_a.py:515:    spell persists. 
dungeonsheets/spells/spells_a.py:516:    A creature that successfully saves against this effect is 
dungeonsheets/spells/spells_a.py:517:    immune to it for 1 minute, after which time it can be affected again.
dungeonsheets/spells/spells_a.py:562:     The portals are two-
dungeonsheets/spells/spells_a.py:566:     
dungeonsheets/spells/spells_a.py:659:    """You and up to eight willing creatures within range project your astral bodies 
dungeonsheets/spells/spells_a.py:660:    into the Astral Plane (the spell fails and the casting is wasted if you are 
dungeonsheets/spells/spells_a.py:661:    already on that plane). 
dungeonsheets/spells/spells_a.py:662:    The material body you leave behind is unconscious and 
dungeonsheets/spells/spells_a.py:663:    in a state of suspended animation; it doesn’t need food or air and doesn’t age. 
dungeonsheets/spells/spells_a.py:666:    Your astral body resembles your mortal form in almost every way, replicating 
dungeonsheets/spells/spells_a.py:667:    your game statistics and possessions. The principal difference is the addition 
dungeonsheets/spells/spells_a.py:668:    of a silvery cord that extends from between your shoulder blades and trails 
dungeonsheets/spells/spells_a.py:669:    behind you, fading to invisibility after 1 foot. This cord is your tether to 
dungeonsheets/spells/spells_a.py:670:    your material body. As long as the tether remains intact, you can find your 
dungeonsheets/spells/spells_a.py:671:    way home. If the cord is cut something that can happen only when an effect 
dungeonsheets/spells/spells_a.py:672:    specifically states that it does your soul and body are separated, killing you 
dungeonsheets/spells/spells_a.py:673:    instantly. 
dungeonsheets/spells/spells_a.py:675:    Your astral form can freely travel through the Astral Plane and can
dungeonsheets/spells/spells_a.py:676:     pass through portals there leading to any other plane. If you enter a new plane
dungeonsheets/spells/spells_a.py:677:     or return to the plane you were on when casting this spell, your body 
dungeonsheets/spells/spells_a.py:678:    and possessions are transported along the silver cord, allowing you to re-enter 
dungeonsheets/spells/spells_a.py:679:    your body as you enter the new plane. Your astral form is a separate 
dungeonsheets/spells/spells_a.py:680:    incarnation. Any damage or other effects that apply to it have no effect on 
dungeonsheets/spells/spells_a.py:681:    your physical body, nor do they persist when you return to it. The spell ends 
dungeonsheets/spells/spells_a.py:682:    for you and your companions when you use your action to dismiss it. When the 
dungeonsheets/spells/spells_a.py:683:    spell ends, the affected creature returns to its physical body, and it awakens. 
dungeonsheets/spells/spells_a.py:686:    The spell might also end early for you or one of your companions. A successful
dungeonsheets/spells/spells_a.py:687:     dispel magic spell used against an astral or physical body ends the spell for 
dungeonsheets/spells/spells_a.py:688:    that creature. If a creature’s original body or its astral form drops to 0 hit 
dungeonsheets/spells/spells_a.py:689:    points, the spell ends for that creature. If the spell ends and the silver cord 
dungeonsheets/spells/spells_a.py:690:    is intact, the cord pulls the creature’s astral form back to its body, ending 
dungeonsheets/spells/spells_a.py:691:    its state of suspended animation. If you are returned to your body prematurely, 
dungeonsheets/spells/spells_a.py:692:    your companions remain in their astral forms and must find their own way back to
dungeonsheets/spells/spells_a.py:693:     their bodies, usually by dropping to 0 hit points.
dungeonsheets/spells/spells_b.py:211:    Clenched Fist 
dungeonsheets/spells/spells_b.py:269:     When a creature enters the wall’s area for the first time on a turn
dungeonsheets/spells/spells_c.py:421:     At any time after the clone matures, if 
dungeonsheets/spells/spells_c.py:534:    Approach The target moves toward you 
dungeonsheets/spells/spells_c.py:537:    Drop The target drops whatever it is holding and then ends its turn.
dungeonsheets/spells/spells_c.py:539:    Flee The target spends its turn moving away from you by the fastest available 
dungeonsheets/spells/spells_c.py:541:    Grovel The target falls prone and then ends its turn.
dungeonsheets/spells/spells_c.py:542:    Halt The target 
dungeonsheets/spells/spells_c.py:743:     1. The creature uses 
dungeonsheets/spells/spells_c.py:747:     
dungeonsheets/spells/spells_c.py:755:     9-10. The creature can act and move 
dungeonsheets/spells/spells_c.py:950:     You choose one the following options for what appears: 
dungeonsheets/spells/spells_c.py:1014:     •  One 
dungeonsheets/spells/spells_c.py:1016:     •  Two fey creatures of challenge 
dungeonsheets/spells/spells_c.py:1018:     •  Four fey creatures of challenge rating 1/2 or lower 
dungeonsheets/spells/spells_c.py:1019:     •  
dungeonsheets/spells/spells_c.py:1064:     the spell ends. The DM answers each question with one word, such as yes, 
dungeonsheets/spells/spells_c.py:1065:    no, maybe, never, irrelevant, or unclear (if the entity doesn’t know 
dungeonsheets/spells/spells_c.py:1098:    Sickness 
dungeonsheets/spells/spells_c.py:1103:    Filth Fever 
dungeonsheets/spells/spells_c.py:1144:    You cast that spell called the 
dungeonsheets/spells/spells_c.py:1145:    contingent spell as part of casting contingency, expending spell slots for 
dungeonsheets/spells/spells_c.py:1249:    Redirect Flow 
dungeonsheets/spells/spells_c.py:1258:    Whirlpool 
dungeonsheets/spells/spells_c.py:1274:    vortex, the object takes 2d8 bludgeoning damage, this damage occurs each round 
dungeonsheets/spells/spells_c.py:1523:    Destroy Water 
dungeonsheets/spells/spells_d.py:66:    illuminate it. 
dungeonsheets/spells/spells_d.py:379:    as "200 feet straight downward" or "upward to the northwest at a 45-degree 
dungeonsheets/spells/spells_d.py:380:    angle, 300 feet". 
dungeonsheets/spells/spells_d.py:405:    """You make yourself – including your clothing, armor, weapons, and other 
dungeonsheets/spells/spells_d.py:406:    belongings on your person – look different until the spell ends or until you 
dungeonsheets/spells/spells_d.py:480:    Break  Enchantment 
dungeonsheets/spells/spells_d.py:486:    Dismissal 
dungeonsheets/spells/spells_d.py:605:     •  50 hit 
dungeonsheets/spells/spells_d.py:607:     •  40 hit points or fewer: deafened and
dungeonsheets/spells/spells_d.py:609:     •  30 hit points or fewer: blinded, deafened, and 
dungeonsheets/spells/spells_d.py:611:     •  20 hit points or fewer: killed instantly 
dungeonsheets/spells/spells_d.py:687:    and general course of action, such as "Attack that creature", "Run over 
dungeonsheets/spells/spells_d.py:688:    there", or "Fetch that object". If the creature completes the order and 
dungeonsheets/spells/spells_d.py:728:    general course of action, such as "Attack that creature", "Run over there", 
dungeonsheets/spells/spells_d.py:729:    or "Fetch that object". If the creature completes the order and doesn’t 
dungeonsheets/spells/spells_d.py:924:    manifest as a golden orb  for clear skies, a cloud for rain, falling snowflakes 
dungeonsheets/spells/spells_d.py:930:    the sound of a small animal, or the faint odor of skunk. The effect  must fit in
dungeonsheets/spells/spells_e.py:270:    Enlarge 
dungeonsheets/spells/spells_e.py:281:    Reduce 
dungeonsheets/spells/spells_e.py:498:    Asleep 
dungeonsheets/spells/spells_e.py:503:    Panicked 
dungeonsheets/spells/spells_e.py:510:    Sickened 
dungeonsheets/spells/spells_f.py:467:    spreads around corners. It ignites flammable objects in the area that aren’t 
dungeonsheets/spells/spells_g.py:493:     Place dancing lights in four corridors. You can designate a simple program that
dungeonsheets/spells/spells_g.py:498:    • Place stinking cloud in two locations. The vapors appear in the 
dungeonsheets/spells/spells_g.py:499:    places you designate; they return within 10 minutes if dispersed by wind while 
dungeonsheets/spells/spells_g.py:500:    guards and wards lasts.
dungeonsheets/spells/spells_g.py:503:    • Place a suggestion in one location. You select an area of up to 5 feet 
dungeonsheets/spells/spells_g.py:506:    suggestion mentally.
dungeonsheets/spells/spells_g.py:508:    The whole warded area radiates magic. A dispel magic cast 
dungeonsheets/spells/spells_g.py:509:    on a specific effect, if successful, removes only that effect.
dungeonsheets/spells/spells_g.py:511:    permanently guarded and warded structure by casting this spell there every 
dungeonsheets/spells/spells_g.py:512:    day for one year.
dungeonsheets/spells/spells_i.py:169:    """You create a magical restraint to hold a creature that you can see within range.
dungeonsheets/spells/spells_i.py:171:    The target must succeed on a Wisdom saving throw or be bound by the spell; if 
dungeonsheets/spells/spells_i.py:172:    it succeeds, it is immune to this spell if you cast it again. While affected by 
dungeonsheets/spells/spells_i.py:173:    this spell, the creature doesn't need to breathe, eat, or drink, and it doesn’t 
dungeonsheets/spells/spells_i.py:174:    age. Divination spells can’t locate or perceive the target.
dungeonsheets/spells/spells_i.py:177:    spell, you choose one of the following forms of imprisonment. 
dungeonsheets/spells/spells_i.py:181:    target is entombed far beneath the earth in a sphere of magical force that is 
dungeonsheets/spells/spells_i.py:182:    just large enough to contain the target. Nothing can pass through the 
dungeonsheets/spells/spells_i.py:183:    sphere, nor can any creature teleport or use planar travel to get into or out of
dungeonsheets/spells/spells_i.py:185:    The special component for this version of the spell is a small mithral orb.
dungeonsheets/spells/spells_i.py:189:    Heavy chains, firmly rooted in the ground, hold the target in place. 
dungeonsheets/spells/spells_i.py:191:    any means until then.
dungeonsheets/spells/spells_i.py:193:    a fine chain of precious metal.
dungeonsheets/spells/spells_i.py:197:    into a tiny demiplane that is warded against teleportation and planar travel. 
dungeonsheets/spells/spells_i.py:198:    The demiplane can be a labyrinth, a cage, a tower, or any similar confined 
dungeonsheets/spells/spells_i.py:199:    structure or area of your choice.
dungeonsheets/spells/spells_i.py:201:    spell is a miniature representation of the prison made from jade.
dungeonsheets/spells/spells_i.py:205:    The target shrinks to a height of 1 inch and is imprisoned inside a 
dungeonsheets/spells/spells_i.py:207:    normally (allowing the target to see out and other creatures to see in), but 
dungeonsheets/spells/spells_i.py:208:    nothing else can pass through, even by means of teleportation or planar travel. 
dungeonsheets/spells/spells_i.py:209:    The gemstone can’t be cut or broken while the spell remains in effect.
dungeonsheets/spells/spells_i.py:211:    special component for this version of the spell is a large, transparent 
dungeonsheets/spells/spells_i.py:212:    gemstone, such as a corundum, diamond, or ruby.
dungeonsheets/spells/spells_i.py:218:    spell consists of rare soporific herbs. 
dungeonsheets/spells/spells_i.py:222:     the spell, in any of its versions, you can specify a condition that will cause 
dungeonsheets/spells/spells_i.py:223:    the spell to end and release the target. The condition can be as specific or as 
dungeonsheets/spells/spells_i.py:224:    elaborate as you choose, but the DM must agree that the condition is reasonable 
dungeonsheets/spells/spells_i.py:225:    and has a likelihood of coming to pass. The conditions can be based on a 
dungeonsheets/spells/spells_i.py:226:    creature’s name, identity, or deity but otherwise must be based on 
dungeonsheets/spells/spells_i.py:227:    observable actions or qualities and not based on intangibles such as level, 
dungeonsheets/spells/spells_i.py:231:    cast as a 9th-level spell, targeting either the prison or the special component 
dungeonsheets/spells/spells_i.py:234:    You can use a particular special component to create only 
dungeonsheets/spells/spells_i.py:235:    one prison at a time. If you cast the spell again using the same component, the 
dungeonsheets/spells/spells_m.py:857:    to the terrain’s true form however, all other elements of the illusion remain, 
dungeonsheets/spells/spells_m.py:858:    so while the creature is aware of the illusion’s presence, the creature can 
dungeonsheets/spells/spells_m.py:859:    still physically interact with the illusion.
dungeonsheets/spells/spells_p.py:69:    exists and comes up with some other explanation for its fallit was pushed, it 
dungeonsheets/spells/spells_p.py:471:     -You create an 
dungeonsheets/spells/spells_p.py:596:    """A shimmering, multicolored plane of light forms a vertical opaque wallup to 90 
dungeonsheets/spells/spells_p.py:597:    feet long, 30 feet high, and 1 inch thickcentered on a point you can seewithin 
dungeonsheets/spells/spells_r.py:161:    d100  Race
dungeonsheets/spells/spells_s.py:96:    Knowledge                 Save 
dungeonsheets/spells/spells_s.py:100:    the target)      +0
dungeonsheets/spells/spells_s.py:105:    Likeness or picture               -2
dungeonsheets/spells/spells_s.py:1510:    turns, if able. 
dungeonsheets/spells/spells_t.py:114:        Mishap  Similar Area  Off Target  On Target
dungeonsheets/spells/spells_t.py:115:    Permanent circle   —       — 
dungeonsheets/spells/spells_t.py:116:            —     01-100
dungeonsheets/spells/spells_t.py:117:    Associated object    —       —       —       01-100
dungeonsheets/spells/spells_t.py:119:    familiar     01-05      06-13     14-24    25-100
dungeonsheets/spells/spells_t.py:120:    Seen casually     01-33      
dungeonsheets/spells/spells_t.py:122:    Viewed once    01-43      44-53     54-73    74-100
dungeonsheets/spells/spells_t.py:124:    Description       01-43      44-53     54-73    74-100
dungeonsheets/spells/spells_t.py:126:          51-100      —      —
dungeonsheets/spells/spells_w.py:480:    3d6 × 10 feet away from it in a random direction.

```

**Comment by pbatroff** [comment]
The above checkin fixes the compile problem, but messes a little bit with some design (e.g. Teleport). But compiling seems to run proper now for me.

**Comment by canismarko** [comment]
I think this has been fixed now with d7737042afb702c06824651eef48c2cb38403e3d and 1186fe7323c77a2c45a823ff999ba72b5d17e710 so I'm closing this issue.

If you run into more problems feel free to re-open.
